modules/ROOT/pages/tgw-about.adoc

```diff
@@ -0,0 +1,49 @@
+= Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+A transit gateway is a network hub that connects your on-premises networks to your VPCs.
+The transit gateway simplifies your network connectivity by functioning
+as a cloud router, managing your VPC connections and requiring only one connection from CloudHub.
+
+Use transit gateway attachments to create a secure connection between your Virtual Private Cloud (VPC) and your private corporate networks.
+You can add multiple transit gateway attachments if required. 
+
+[NOTE]
```

Comment:
Does this need to be emphasized in a note, or can we just tag it on to the previous sentence?
```suggestion
You can add multiple transit gateway attachments, depending on your account entitlements....
```